src/server/apps/worldserver/Main.cpp

```diff
@@ -391,6 +391,15 @@ int main(int argc, char** argv)
 
     sScriptMgr->OnStartup();
 
+// Be kind and warn people of EOL deprecation :)
+#if !defined(MARIADB_VERSION_ID)
+    if (MySQL::GetLibraryVersion() < 80000)
+        LOG_WARN("server", "WARNING: You are using MySQL version 5.7 which is soon EOL!\nThis version will be deprecated. Consider upgrading to MySQL 8.0 or 8.1!");
```

Comment:
```suggestion
    if (MySQL::GetLibraryVersion() < 80000)
    {
            LOG_WARN("server", "WARNING: You are using MySQL version 5.7 which is soon EOL!\nThis version will be deprecated. Consider upgrading to MySQL 8.0 or 8.1!");
    }
```
Please read the wiki article regarding cpp code standards, specifically the section on [brackets](https://www.azerothcore.org/wiki/cpp-code-standards#brackets).

Reply:
> Brackets should be refrained from using for if statements that are followed by one line